jenkins/scripts/node-test-commit-test.sh

```diff
@@ -0,0 +1,43 @@
+#/bin/bash
+
+rm -rf build
+git clone https://github.com/nodejs/build.git
+
+. ./build/jenkins/scripts/node-test-commit-pre.sh
+
+if test $IGNORE_FLAKY_TESTS = "true"
+then
+  FLAKY_TESTS_MODE=dontcare
+else
+  FLAKY_TESTS_MODE=run
+fi
+echo FLAKY_TESTS_MODE=$FLAKY_TESTS_MODE
+
+echo $NODE_NAME
+
+. ./build/jenkins/scripts/select-compiler.sh
+
+if [ -x "$(command -v make)" ]
```

Comment:
Does this work on AIX? I don't know how the community AIX systems are set up but (on the internal IBM ones I have access to) [`make` exists](https://www.ibm.com/support/knowledgecenter/en/ssw_aix_61/com.ibm.aix.cmds3/make.htm) but is not `GNU make` that is required to build Node.js. Maybe test for `gmake` and default to `make` if not found?